vllm/utils.py

```diff
@@ -31,7 +31,7 @@
     "half": torch.half,
     "bfloat16": torch.bfloat16,
     "float": torch.float,
-    "fp8": torch.uint8,
+    "fp8": torch.float8_e4m3fn,
```

Comment:
why is that needed? are we sure we're not breaking compatibility with this?

Reply:
added hf8 for float8_e4m3fn